docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -1,45 +1,33 @@
 
-![Quality settings for pull requests](images/quality-settings-pull-requests.png)
+![Quality gates](images/quality-settings-gates.png)
 
--   **New issues are over:** Pull requests are marked not up to standards if the number of issues introduced that have at least the specified severity level is bigger than the set value. The default value is 0.
--   **New security issues are over:** Pull requests are marked not up to standards if the number of security issues introduced is bigger than the set value. By default, this setting is off.
--   **Complexity is over:** Pull requests are marked not up to standards if the introduced complexity is bigger than the set value. By default, this setting is off.
--   **Duplication is over:** Pull requests are marked not up to standards if the number of clones introduced is bigger than the set value. By default, this setting is off.
--   **Coverage variation is under:** Pull requests are marked not up to standards if the coverage introduced is smaller than the set value. This value refers to the variation of coverage on your pull request. For example, if your repository has 90% coverage and you want to make sure that this value doesn't decrease, set the value to 0%. By default, this setting is off. The maximum value is 1%.
+-   **New issues are over:** Pull requests or commits are marked not up to standards if the number of issues introduced that have at least the specified severity level is higher than the set value. The default value is 0.
+-   **New security issues are over:** Pull requests or commits are marked not up to standards if the number of security issues introduced is higher than the set value. By default, this setting is off.
+-   **Complexity is over:** Pull requests or commits are marked not up to standards if the introduced complexity is higher than the set value. By default, this setting is off.
+-   **Duplication is over:** Pull requests or commits are marked not up to standards if the number of clones introduced is higher than the set value. By default, this setting is off.
+-   **Coverage variation is under:** Pull requests or commits are marked not up to standards if the coverage introduced is smaller than the set value. This value refers to the variation of coverage on your pull requests or commits. For example, if your repository has 90% coverage and you want to make sure that this value doesn't decrease, set the value to 0%. By default, this setting is off. The maximum value is 1%.
 
-## Repository
+## Goals
 
-These settings configure quality thresholds for the repository, as displayed on the [Repository Dashboard](../repositories/repository-dashboard.md).
+These settings configure the quality standard thresholds for your repository that's displayed as a dashed line on the [quality evolution chart](../repositories/repository-dashboard.md#quality-evolution-chart), and which files Codacy considers complex and duplicated.
```

Comment:
@prcr It's not very clear to me, feels like that there's too many concepts in one sentence. Maybe consider to breakdown more the structure of the paragraph to be very clear that the quality goals is where the user can define their thresholds for X and Z, and that those will support the user to track progress by becoming visible in their dashboard. (Let me know if I wasn't clear here)

Reply:
Good feedback, indeed this can be simplified and improved! :bow: